.vscode/settings.json

```diff
@@ -56,5 +56,9 @@
         // Match what black does.
         "--max-line-length=88"
     ],
-    "typescript.preferences.importModuleSpecifier": "relative"
+    "typescript.preferences.importModuleSpecifier": "relative",
```

Comment:
Unit tests will show up in the test explorer now, just copied these settings from the `build/.mocha.unittests.ts.json` file